src/features/lightspeed/handleApiError.ts

```diff
@@ -20,38 +20,42 @@ export function retrieveError(err: AxiosError): string {
         return "Bad Request response. Please try again.";
       }
     } else if (err?.response?.status === 403) {
```

Comment:
The change is not as bad as it looks.

I simply check for a CloudFront error otherwise let the existing 403 handling run:
```
...
    } else if (err?.response?.status === 403) {
      if((err?.response?.headers["server"] || "").toLowerCase() === "cloudfront") {
        return `CloudFront rejected the request. Please contact your administrator.`;
      } else {
        const responseErrorData = <AxiosError<{ message?: string }>>(
          err?.response?.data
        );
...
```